spryker/setup_suite.sh

```diff
@@ -40,6 +40,7 @@ fi
 ##sudo touch ~/.composer/auth.json
 ##sudo echo '{ "http-basic": {}, ' >  ~/.composer/auth.json
 ##sudo echo "\"github-oauth\": { \"github.com\": \"$GITHUB_TOKEN\"}}" >>  ~/.composer/auth.json
+composer config -g github-oauth.github.com $GITHUB_TOKEN
```

Comment:
Do we need to run the command if $GITHUB_TOKEN is empty?

Reply:
Updated with the next commit